learnosity_sdk/request.py

```diff
@@ -219,3 +222,62 @@ def set_service_options(self):
     def hash_list(self, l):
         "Hash a list by concatenating values with an underscore"
         return hashlib.sha256("_".join(l).encode('utf-8')).hexdigest()
+
+
+class DataApi(object):
+
+    def request(self, endpoint, security_packet,
+                secret, request_packet, action):
+        """Make a single request to Data API"""
```

Comment:
We should add a longer docstring with the various parameters and the type of the return value, so it can be investigated with `help()`.

Same for all other methods.